packages/block-library/src/post-time-to-read/edit.js

```diff
@@ -166,33 +161,49 @@ function PostTimeToReadEdit( { attributes, setAttributes, context } ) {
 					resetAll={ () => {
 						setAttributes( {
 							displayAsRange: true,
-							showTimeToRead: true,
-							showWordCount: false,
+							displayMode: 'time',
 						} );
 					} }
 					dropdownMenuProps={ dropdownMenuProps }
 				>
 					<ToolsPanelItem
-						label={ __( 'Show time to read' ) }
-						hasValue={ () => ! showTimeToRead }
+						label={ __( 'Display options' ) }
+						hasValue={ () => displayMode !== 'time' }
```

Comment:
```suggestion
						isShownByDefault
						label={ __( 'Display options' ) }
						hasValue={ () => displayMode !== 'time' }
```

Nit: My understanding is that for now all controls should be visible by default. See: https://github.com/WordPress/gutenberg/issues/67813#issuecomment-2536574349